Assets/Scripts/Item.cs

```diff
@@ -29,9 +38,19 @@ public class Item : MonoBehaviour {
   void Start() {
     originalScale = transform.lossyScale;
     originalLayer = gameObject.layer;
+    if (Rigidbody) originalMass = Rigidbody.mass;
+
     foreach(Transform childTransform in transform) {
       children.Add(childTransform.gameObject);
     }
+    allColliders = new List<Collider>(GetComponentsInChildren<Collider>());
+
+    itemBounds = allColliders[0].bounds;
+    for(int i = 1; i < allColliders.Count; i++){
+      itemBounds.Encapsulate(allColliders[i].bounds);
+    }
+
+    structureBounds = GameObject.Find("Structure").GetComponent<Collider>().bounds;    
```

Comment:
I think adding a collider field that's also exposed in the inspector for this "Structure" collider would be a bit more efficient / harder to accidentally forget when setting up an item object.

Reply:
Oh honestly I forgot that was there. I no longer use the structureBounds variable. I'll just remove it